packages/vitest/src/node/cli.ts

```diff
@@ -48,6 +48,7 @@ cli
   .option('--no-color', 'Removes colors from the console output')
   .option('--inspect', 'Enable Node.js inspector')
   .option('--inspect-brk', 'Enable Node.js inspector with break')
+  .option('--test-timeout', 'Default timeout of a test in milliseconds')
```

Comment:
1. Does `cac` know that it's a number?
2. Should we specify default in brackets like we do for other options?

Reply:
2: Done

1: It works when a number is passed:
```
$ pnpm run test --test-timeout 1
> vitest run "--test-timeout" "1"

...

 FAIL  test/animals.test.ts > Dog says woof
Error: Test timed out in 1ms.
```

But it actually accepts anything:
```
$ pnpm run test --test-timeout some-text
> vitest run "--test-timeout" "some-text"

...

 FAIL  test/animals.test.ts > Dog says woof
Error: Test timed out in some-textms.
```

But even the existing `--shard` option accepts incorrect type so I guess there is no checking for types in place:
```
$ pnpm run test --shard 1
> vitest run "--shard" "1"

TypeError: options.shard.split is not a function or its return value is not iterable
 ELIFECYCLE  Test failed. See above for more details.
```